### PR TITLE
Feature/sfat 317 log retention

### PR DESCRIPTION
**Pull request by adrianmilne**
Made ECS & API log retention configurable and set to 30 days for NFT/PPD/PRD (which aligns it with CloudTrail log retention)

### CODE REVIEW

File: terraform/modules/configs/deploy-all/variables.tf [hunk 1]
```diff
@@ -23,7 +23,7 @@ variable "ecr_image_id_decision_tree" {
 
 variable "ecr_image_id_decision_tree_db" {
   type    = string
-  default = "8a8c302-candidate"
+  default = "d39cc53-snapshot"
```
Comment (tommyb82) [inline terraform/modules/configs/deploy-all/variables.tf]: Snapshot version crept into this PR for decision tree DB service - remove

**Review by tommyb82 — CHANGES_REQUESTED** [review]
Changes look good - just that snapshot image version to remove 👍

File: terraform/modules/configs/deploy-all/variables.tf (hunk 1 above)
Comment (adrianmilne) [reply terraform/modules/configs/deploy-all/variables.tf]: Ahh - sorry - will update (actually need to deploy a new one - so 2 birds with one PR)

Comment (adrianmilne) [reply terraform/modules/configs/deploy-all/variables.tf]: Updated now 👍

**Review by tommyb82 — COMMENTED** [review]
Magnifique! 👍

**Comment by adrianmilne** [comment]
> Magnifique! 👍

Est-il approuvé? Ok to merge?...

**Review by tommyb82 — APPROVED** [review]

**Pull request merged**